kubeflow/hack/create_context.sh

```diff
@@ -6,24 +6,17 @@
 # PROJECT=<PROJECT> REGION=<region> NAME=<NAME>
 #
 # TODO(jlewi): Support zonal clusters as well
-set -x 
+set -x
 
-# Default namespace to kubeflow
-NAMESPACE=${NAMESPACE:-kubeflow}
-echo Checking if context ${NAME} exists 
-
-kubectl config use-context ${NAME}
-
-RESULT=$?
-
-if [ ${RESULT} -eq 0 ]; then
-echo kubeconfig context ${NAME} already exists
-exit 0
-fi
+# Delete kubeconfig context if exist already
+kubectl config delete-context ${NAME} || echo "Ignore error - cannot delete context ${NAME}"
```

Comment:
sounds good

nit: What do you think about? or do you have any other ideas?
`Context ${NAME} doesn't exist, so we will create a new one.`

The key information is why the context doesn't exist isn't a problem.

Reply:
Does a user really need to get the details of what is going to happen behind the scenes?
OR
Notifying a user about intentionally ignoring an error is sufficient?

Also, I feel maybe we should not use `so we will create a new one`. Because who is we does not seem clear.